app/konnect/network.md

```diff
@@ -43,16 +43,16 @@ add the following hostnames to the firewall allowlist:
 * `<region>.api.konghq.com`: The {{site.konnect_short_name}} API.
   Necessary if you are using decK in your workflow, decK uses this API to access and apply configurations.
   Region can be `us`, `eu`, or `global`.
-* `<runtime-group-id>.<region>.cp0.konghq.com`: Handles configuration for a control plane.
+* `<controlPlaneId>.<region>.cp0.konghq.com`: Handles configuration for a control plane.
   Data plane nodes connect to this host to receive configuration updates.
   This hostname is unique to each organization and control plane.
-* `<runtime-group-id>.<region>.tp0.konghq.com`: Gathers telemetry data for a control plane.
+* `<controlPlaneId>.<region>.tp0.konghq.com`: Gathers telemetry data for a control plane.
   This hostname is unique to each organization and control plane.
 
 You can find the configuration and telemetry hostnames through the Gateway Manager:
 
 1. Open a control plane.
-2. Click **Add runtime instance**.
+2. Click **Add data plane**.
```

Comment:
Well this is just confusing now. Are they using "data plane" in the UI? They shouldn't be.